packages/grafana-data/src/types/navModel.ts

```diff
@@ -28,6 +28,7 @@ export interface NavLinkDTO {
   emptyMessageId?: string;
   // The ID of the plugin that registered the page (in case it was registered by a plugin, otherwise left empty)
   pluginId?: string;
+  isCreateAction?: boolean;
```

Comment:
definitely yes to adding the description, i'll do that now. 👍 

the name i'm a bit more hesitant on 😛 trying to avoid tying it too tightly to the concept of quick add. say tomorrow we decide we don't want a quick add button... this prop then doesn't make sense anymore. but distinguishing between create actions + other pages will probably always be useful in some way.